backend/backend-python/model/gptoss.py

```
@@ -836,7 +845,7 @@ def forward(
         kv_last_page_lens: torch.Tensor,
         custom_mask: torch.Tensor,
         single_token_inference_mode: bool,
```

Ruff (0.13.1)

Unused method argument: `single_token_inference_mode`

(ARG002)